google/cloud/aiplatform/jobs.py

```diff
@@ -46,19 +46,10 @@ class _Job(base.AiPlatformResourceNoun):
     client_class = job_service.JobServiceClient
     _is_client_prediction_client = False
 
-    @property
-    @abc.abstractclassmethod
-    def _getter_method(cls) -> str:
-        """Name of getter method of Job subclass, i.e. 'get_custom_job' for CustomJob"""
-        pass
-
-    def _get_job(self, job_name: str):
-        """Returns GAPIC service representation of Job subclass resource"""
-        return getattr(self.api_client, self._getter_method)(name=job_name)
-
     def __init__(
         self,
-        valid_job_name: str,
+        job_name: str,
+        job_type: str,
```

Comment:
`job_type` seems unnecessary because this will always be subclassed the that value equals `cls._resource_name`